includes/general.php

```diff
@@ -2448,6 +2452,7 @@ function pods_no_conflict_on( $object_type = 'post', $object = null ) {
 				$no_conflict['filter'] = array_merge( $no_conflict['filter'], array(
 					array( 'add_comment_metadata', array( PodsInit::$meta, 'add_comment_meta' ), 10, 5 ),
 					array( 'update_comment_metadata', array( PodsInit::$meta, 'update_comment_meta' ), 10, 5 ),
+					array( 'update_comment_metadata_by_id', array( PodsInit::$meta, 'update_comment_meta_by_id' ), 10, 4 ),
```

Comment:
* :no_entry_sign: Short array syntax must be used to define arrays
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 8abcaa39a6b73360b20438ce9230803e</sub>